account_invoice_import/wizard/account_invoice_import.py

```diff
@@ -423,12 +435,23 @@ def create_invoice(self):
         invoice = self._create_invoice(parsed_inv)
         invoice.message_post(_(
             "This invoice has been created automatically via file import"))
+        if invoice.amount_total != parsed_inv.get('amount_total'):
```

Comment:
Please use a dedicated float comparing method from the currency object.

Reply:
Done!

Reply:
Thanks, but you have the currency available from the invoice and it provides its own float compare method. That can actually make a difference if there are tax rounding differences.

Reply:
Done!